.github/workflows/flownet.yml

```diff
@@ -88,6 +88,7 @@ jobs:
           flownet ahm ./config/assisted_history_matching.yml ./some_ahm_run --update-config ./ci_config/assisted_history_matching.yml
```

Comment:
Can we add one line here where the new functionality is used? Then we are sure the added functionality runs :green_heart: :slightly_smiling_face: